etc/testing/start_postgres.sh

```diff
@@ -1,12 +1,12 @@
 #!/bin/sh
 
-if ! docker ps | grep -q postgres
+if ! docker ps | grep -q postgres_test
 then
     echo "starting postgres..."
-    docker run -d \
-    -e POSTGRES_DB=pgc \
+    docker run -d --rm \
+    --name postgres_test \
     -e POSTGRES_HOST_AUTH_METHOD=trust \
-    -p 32228:5432 \
+    -p 5432:5432 \
```

Comment:
This is kind of annoying if you have an unrelated postgres instance using the default port. It might be better to just have our own postgres_test port.

Reply:
The check just looks for a container called postgres.  I think sharing the default port in this case will make problems with multiple postgres's more obvious.